app/views/customer_service/index.html.erb

```diff
@@ -0,0 +1,332 @@
+<%- @page_title = 'Customer Service' -%>
+
+<!-- Don't automatically turn FAX numbers into links -->
+<meta name = "format-detection" content = "telephone=no">
```

Comment:
This was a new one to me. 👍 

I think since generally we're wrapping telephone numbers in `<a>` already, we should move this [to the layout](https://github.com/deptofdefense/move.mil/blob/master/app/views/layouts/application.html.erb) between lines 9 and 10:

```erb
<meta name="application-name" content="<%= t('site.name') %>">
<meta name="format-detection" content="telephone=no">
<meta name="theme-color" content="#d6d7d9">
```

Make sure to remove the spacing around the equals signs, too.